restli-example-client/src/main/java/com/linkedin/restli/example/photos/AlbumsCreateRequestBuilder.java

```diff
@@ -0,0 +1,46 @@
+
+package com.linkedin.restli.example.photos;
+
+import com.linkedin.data.schema.validation.ValidationResult;
+import com.linkedin.restli.client.RestliRequestOptions;
+import com.linkedin.restli.client.base.CreateIdRequestBuilderBase;
+import com.linkedin.restli.common.ResourceMethod;
+import com.linkedin.restli.common.ResourceSpec;
+import com.linkedin.restli.common.RestConstants;
+import com.linkedin.restli.common.validation.RestLiDataValidator;
+import com.linkedin.restli.example.Album_$Versioned;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import javax.annotation.Generated;
+
+
+@Generated(value = "com.linkedin.pegasus.generator.JavaCodeUtil", comments = "Rest.li Request Builder")
+public class AlbumsCreateRequestBuilder
+    extends CreateIdRequestBuilderBase<Long, Album_$Versioned, AlbumsCreateRequestBuilder>
+{
```

Comment:
Since we are already generating code. Could we generate a specific builder for each version instead of relying on inheritance? This would allow us to support versioning on the method level in the future.